frontend-vue/src/pages/domains/Root.vue

```diff
@@ -97,4 +100,10 @@ const tabPanelViews: TabPanelOption[] = [
 function onTabChange(newSelectedTab: number): void {
   queryParamType.value = tabOptions[newSelectedTab].id;
 }
+
+watchEffect(() => {
```

Comment:
This `watchEffect` can be removed if we separate it into it's own component.

Reply:
Removed.